### ISSUE TITLE
Update Fedora install doc for F27

### ISSUE THREAD

**Issue by jodh-intel** [clearcontainers/runtime#882]
Now that Docker have finally released the following:

- https://download.docker.com/linux/fedora/27/x86_64/stable/Packages/docker-ce-17.12.0.ce-1.fc27.x86_64.rpm

... we need to update and re-test the installation doc for Fedora 27:

- https://github.com/clearcontainers/runtime/blob/master/docs/fedora-installation-guide.md

**Comment by jodh-intel** [comment]
Note that since F24 and F25 have **both** now gone end-of-life according to https://fedoraproject.org/wiki/End_of_life, we need to:

- Remove those two versions from the install doc.
- Remove the OBS binaries for those old distros I think.
- Remove any F24/F25 specific code from the test repos.

/cc @gorozco1, @erick0z, @chavafg, @grahamwhaley, @sameo.

**Comment by GabyCT** [comment]
@jodh-intel , I noticed that there are not OBS packages for Fedora 27 at http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3

**Comment by jodh-intel** [comment]
@GabyCT - @erick0z has now created the F27 packages so this should unblock you.

**Comment by GabyCT** [comment]
@jodh-intel and @erick0z , we have an issue when trying to install the OBS packages in F27 (clean environment) , as qemu is not installed
$ docker run -ti busybox sh
docker: Error response from daemon: OCI runtime create failed: /usr/share/defaults/clear-containers/configuration.toml: file /usr/bin/qemu-lite-system-x86_64 does not exist: unknown.

This can be viewed here :

$ source /etc/os-release

$ sudo -E VERSION_ID=$VERSION_ID dnf config-manager --a                                                                                                              dd-repo \
> http://download.opensuse.org/repositories/home:/clearcontainers:/clear-contain                                                                                                              ers-3/Fedora\_$VERSION_ID/home:clearcontainers:clear-containers-3.repo
Adding repo from: http://download.opensuse.org/repositories/home:/clearcontainer                                                                                                              s:/clear-containers-3/Fedora_27/home:clearcontainers:clear-containers-3.repo

$ sudo -E dnf -y install cc-runtime cc-proxy cc-shim
....
Key imported successfully
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                                                                                                      1/1
  Installing       : python2-libs-2.7.14-4.fc27.x86_64                                                                                                                                   1/22
  Running scriptlet: python2-libs-2.7.14-4.fc27.x86_64                                                                                                                                   1/22
  Installing       : python2-pip-9.0.1-14.fc27.noarch                                                                                                                                    2/22
  Installing       : python2-setuptools-37.0.0-1.fc27.noarch                                                                                                                             3/22
  Installing       : python2-2.7.14-4.fc27.x86_64                                                                                                                                        4/22
  Installing       : cc-proxy-config-3.0.12+git.3c6daa6-17.1.x86_64                                                                                                                      5/22
  Installing       : cc-runtime-config-3.0.12+git.d9f04c9-17.1.x86_64                                                                                                                    6/22
  Installing       : cc-runtime-bin-3.0.12+git.d9f04c9-17.1.x86_64                                                                                                                       7/22
  Installing       : cc-proxy-bin-3.0.12+git.3c6daa6-17.1.x86_64                                                                                                                         8/22
  Installing       : audit-libs-python-2.8.2-1.fc27.x86_64                                                                                                                               9/22
  Installing       : libselinux-python-2.7-3.fc27.x86_64                                                                                                                                10/22
  Installing       : libsemanage-python-2.7-2.fc27.x86_64                                                                                                                               11/22
  Installing       : python2-IPy-0.81-20.fc27.noarch                                                                                                                                    12/22
  Installing       : python2-enum34-1.1.6-3.fc27.noarch                                                                                                                                 13/22
  Installing       : setools-python-4.1.1-3.fc27.x86_64                                                                                                                                 14/22
  Installing       : policycoreutils-python-2.7-4.fc27.x86_64                                                                                                                           15/22
  Installing       : clear-containers-selinux-0.1-4.1.noarch                                                                                                                            16/22
  Running scriptlet: clear-containers-selinux-0.1-4.1.noarch                                                                                                                            16/22
  Installing       : cc-proxy-3.0.12+git.3c6daa6-17.1.x86_64                                                                                                                            17/22
  Installing       : cc-shim-bin-3.0.12+git.d01f9a7-17.1.x86_64                                                                                                                         18/22
  Installing       : cc-shim-3.0.12+git.d01f9a7-17.1.x86_64                                                                                                                             19/22
  Installing       : linux-container-4.9.60-82.1.x86_64                                                                                                                                 20/22
  Installing       : clear-containers-image-19790-43.1.x86_64                                                                                                                           21/22
  Installing       : cc-runtime-3.0.12+git.d9f04c9-17.1.x86_64                                                                                                                          22/22
  Running scriptlet: cc-runtime-3.0.12+git.d9f04c9-17.1.x86_64                                                                                                                          22/22
Running as unit: run-r7840fa3b1fcb4138badca6ac55f909d3.service
  Verifying        : cc-runtime-3.0.12+git.d9f04c9-17.1.x86_64                                                                                                                           1/22
  Verifying        : cc-proxy-3.0.12+git.3c6daa6-17.1.x86_64                                                                                                                             2/22
  Verifying        : cc-shim-3.0.12+git.d01f9a7-17.1.x86_64                                                                                                                              3/22
  Verifying        : cc-runtime-bin-3.0.12+git.d9f04c9-17.1.x86_64                                                                                                                       4/22
  Verifying        : cc-runtime-config-3.0.12+git.d9f04c9-17.1.x86_64                                                                                                                    5/22
  Verifying        : clear-containers-image-19790-43.1.x86_64                                                                                                                            6/22
  Verifying        : linux-container-4.9.60-82.1.x86_64                                                                                                                                  7/22
  Verifying        : cc-proxy-bin-3.0.12+git.3c6daa6-17.1.x86_64                                                                                                                         8/22
  Verifying        : cc-proxy-config-3.0.12+git.3c6daa6-17.1.x86_64                                                                                                                      9/22
  Verifying        : clear-containers-selinux-0.1-4.1.noarch                                                                                                                            10/22
  Verifying        : cc-shim-bin-3.0.12+git.d01f9a7-17.1.x86_64                                                                                                                         11/22
  Verifying        : policycoreutils-python-2.7-4.fc27.x86_64                                                                                                                           12/22
  Verifying        : audit-libs-python-2.8.2-1.fc27.x86_64                                                                                                                              13/22
  Verifying        : libselinux-python-2.7-3.fc27.x86_64                                                                                                                                14/22
  Verifying        : libsemanage-python-2.7-2.fc27.x86_64                                                                                                                               15/22
  Verifying        : python2-IPy-0.81-20.fc27.noarch                                                                                                                                    16/22
  Verifying        : setools-python-4.1.1-3.fc27.x86_64                                                                                                                                 17/22
  Verifying        : python2-2.7.14-4.fc27.x86_64                                                                                                                                       18/22
  Verifying        : python2-libs-2.7.14-4.fc27.x86_64                                                                                                                                  19/22
  Verifying        : python2-enum34-1.1.6-3.fc27.noarch                                                                                                                                 20/22
  Verifying        : python2-pip-9.0.1-14.fc27.noarch                                                                                                                                   21/22
  Verifying        : python2-setuptools-37.0.0-1.fc27.noarch                                                                                                                            22/22

Installed:
  cc-proxy.x86_64 3.0.12+git.3c6daa6-17.1     cc-runtime.x86_64 3.0.12+git.d9f04c9-17.1      cc-shim.x86_64 3.0.12+git.d01f9a7-17.1        audit-libs-python.x86_64 2.8.2-1.fc27
  cc-proxy-bin.x86_64 3.0.12+git.3c6daa6-17.1 cc-proxy-config.x86_64 3.0.12+git.3c6daa6-17.1 cc-runtime-bin.x86_64 3.0.12+git.d9f04c9-17.1 cc-runtime-config.x86_64 3.0.12+git.d9f04c9-17.1
  cc-shim-bin.x86_64 3.0.12+git.d01f9a7-17.1  clear-containers-image.x86_64 19790-43.1       clear-containers-selinux.noarch 0.1-4.1       libselinux-python.x86_64 2.7-3.fc27
  libsemanage-python.x86_64 2.7-2.fc27        linux-container.x86_64 4.9.60-82.1             policycoreutils-python.x86_64 2.7-4.fc27      python2.x86_64 2.7.14-4.fc27
  python2-IPy.noarch 0.81-20.fc27             python2-enum34.noarch 1.1.6-3.fc27             python2-libs.x86_64 2.7.14-4.fc27             python2-pip.noarch 9.0.1-14.fc27
  python2-setuptools.noarch 37.0.0-1.fc27     setools-python.x86_64 4.1.1-3.fc27

Complete!

**Comment by erick0z** [comment]
@GabyCT Yep, for some reason qemu-lite is not being installed, although it is listed as dependency. I installed `qemu-lite` with `dnf install qemu-lite` and works as expected. Anyway, let me dig more into the issue, qemu-lite should be installed automatically. Thanks!

**Comment by jodh-intel** [comment]
Confirmed - I see the same behaviour.
